ang/civicase/ActivityActions.js

```diff
@@ -0,0 +1,46 @@
+(function ($, _, angular) {
+  var module = angular.module('civicase');
+
+  module.directive('civicaseActivityActions', function () {
+    return {
+      scope: {
+        selectedActivities: '='
+      },
+      controller: civicaseActivityActionsController,
+      templateUrl: '~/civicase/ActivityActions.html',
+      restrict: 'A'
+    };
+
+    function civicaseActivityActionsController ($scope, crmApi) {
+      var ts = $scope.ts = CRM.ts('civicase');
+
+      /**
+       * Delete an activity
+       *
+       * @param {object} activity
+       * @param {jQuery} dialog - the dialog which should be closed once deletion is over
+       */
+      $scope.deleteActivity = function (activities, dialog) {
+        CRM.confirm({
+          title: ts('Delete Activity'),
+          message: ts('Permanently delete %1 activit%2?', {1: activities.length, 2: activities.length > 1 ? 'ies' : 'y'})
+        }).on('crmConfirm:yes', function () {
+          var calls = [];
```

Comment:
Can we call it `apiCalls`. `calls` sounds vague. Also let's update the appearance of this in following lines of code.